posthog/async_migrations/utils.py

```diff
@@ -52,10 +52,10 @@ def process_error(migration_instance: AsyncMigration, error: Optional[str]):
     attempt_migration_rollback(migration_instance)
 
 
-def trigger_migration(migration_instance: AsyncMigration, fresh_start: bool = True, countdown: int = 0):
```

Comment:
this needed some more thinking through, so I've removed it for now. we don't auto start migrations yet anyway so this isn't a problem